pkg/transform/kubernetes/kubernetes.go

```diff
@@ -158,9 +171,12 @@ func (t *Transformer) CreateDeployments(o *object.Service) ([]runtime.Object, er
 			},
 			Template: api_v1.PodTemplateSpec{
 				ObjectMeta: api_v1.ObjectMeta{
-					Labels: map[string]string{
-						"service": o.Name,
-					},
+					Labels: *util.MergeMaps(
```

Comment:
I would swap order of parameters here.
```
*util.MergeMaps(
  &serviceLabels,
  &map[string]string{"service": o.Name}
),
```

Map with "service" label should be the last as it should never be overwritten.
Its used as selector for Service and other stuff.

I would also add comment explaining that it has to be last and why is that.

Same thing in other places setting Labels